src/import-data/ImportDataDestination.ts

```diff
@@ -220,14 +219,20 @@ export const ImportDataDestination = (props: ImportDataDestinationProps): ReactN
             }),
           ]),
       ]),
-      importMayTakeTime && div({ style: { marginTop: '0.5rem', lineHeight: '1.5' } }, [importMayTakeTimeMessage]),
-      !!selectedWorkspace &&
-        h(Fragment, [
-          h(WorkspacePolicies, { workspace: selectedWorkspace }),
-          !!isProtectedData &&
-            isAzureWorkspace(selectedWorkspace) &&
-            div([p(['Importing this data may add:']), ul([li(['Additional access controls'])])]),
```

Comment:
Since this only ever has a single element, I didn't see any reason to make it a list.